devcontainer.md

```diff
@@ -0,0 +1,111 @@
+For information on running demos and tests using provided shell scripts, see [DevReadMe](/DevReadMe.md) readme.
+
+# ACA-Py Development with Dev Container
+The following guide will get you up and running and developing/debugging ACA-Py as quickly as possible. 
+We provide a [`devcontainer`](https://containers.dev) and will use [`VS Code`](https://code.visualstudio.com) to illustrate.
+
+By no means is ACA-Py limited to these tools; they are merely examples.  
+
+## Further Reading and Links
+
+* Development Containers (devcontainers): [https://containers.dev](https://containers.dev)
+* Visual Studio Code: [https://code.visualstudio.com](https://code.visualstudio.com)
+* Dev Containers Extension: [marketplace.visualstudio.com](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+* Docker: [https://www.docker.com](https://www.docker.com)
+* Docker Compose: [https://docs.docker.com/compose/](https://docs.docker.com/compose/)
+
+
+## Devcontainer
+
+> What are Development Containers?
+> 
+> A Development Container (or Dev Container for short) allows you to use a container as a full-featured development environment. It can be used to run an application, to separate tools, libraries, or runtimes needed for working with a codebase, and to aid in continuous integration and testing. Dev containers can be run locally or remotely, in a private or public cloud.
+
+see [https://containers.dev](https://containers.dev).
+
+In this guide, we will use [Docker](https://www.docker.com) and [Visual Studio Code](https://code.visualstudio.com) with the [Dev Containers Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed, please set your machine up with those. As of writing, we used the following: 
+
+- Docker Version: 20.10.24
+- VS Code Version: 1.79.0
+- Dev Container Extension Version: v0.295.0
+
+### Open ACA-Py in the devcontainer
+
+There are 2 ways to open ACA-Py into the devcontainer and start developing.
+
+1. Open Visual Studio Code, and use the Command Palette to open the devcontainer.
+    
+    ```
+    > Dev Containers: Open Folder in Container...
```

Comment:
I would suggest adding what folder to select.  I think you select the root of ACA-Py, vs. `.devcontainer` which is kind of what I expected.  Real devs might have a better intuition...

Reply:
done.